src/client/govuk-frontend.d.ts

```diff
@@ -1,7 +1,13 @@
 /* eslint-disable lines-between-class-members */
+// eslint-disable-next-line max-classes-per-file
 declare module 'govuk-frontend' {
-  export class Header<E extends HTMLElement = HTMLElement> {
-    constructor(el: E)
+  class Accordion {
+    constructor(el: HTMLElement)
```

Comment:
Tried a few different ways to type `el` as `HTMLDivElement` but didn't find something that worked...